ui/pages/Login/components/GoogleAuth.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+
+import { ButtonLink } from 'shared/components/StyledComponents'
+import MessagesPanel from 'shared/components/form/MessagesPanel'
+import DataLoader from 'shared/components/DataLoader'
+import { getGoogleAuthIsLoading, getGoogleAuthIsLoadingError } from 'redux/selectors'
+import { loadGoogleAuthResult } from '../reducers'
+
+const GoogleAuth = React.memo(({ location, load, loading, errorMessage }) => {
+  const errMsgs = [errorMessage]
+  return (
+    <div>
+      <DataLoader contentId={location} content={location} load={load} loading={loading}>
+        <MessagesPanel errors={errMsgs} />
```

Comment:
`errorMessage` is null/undefined this will still render an error message panel which I don't think is what you want.I think this should be `errorMessage && [errorMessage]`